src/composables/PluginManager.js

```diff
@@ -154,3 +154,22 @@ export function getPlugins() {
 export function getPluginByName(name) {
   return instanciatePlugins.find((p) => p.name === name);
 }
+
+/**
+ * Delete a component from a tree of components
+ *
+ * @param {Object} component - component to find and remove.
```

Comment:
You don't receive a component here. Please rename to Event and update the documenation like:
```
* @param {Object} event - Event that indicate all details of componnet to remve
* @param {String} event.id - Component id to remove
```

Reply:
Done (better)